proxy/director.go

```diff
@@ -22,24 +22,17 @@ import (
 	"time"
 )
 
-const (
-	// amount of time an endpoint will be held in a failed
-	// state before being reconsidered for proxied requests
-	endpointFailureWait = 5 * time.Second
-
-	// how often the proxy will attempt to refresh its set of endpoints
-	refreshEndpoints = 30 * time.Second
-)
-
-func newDirector(urlsFunc GetProxyURLs) *director {
+func newDirector(urlsFunc GetProxyURLs, failureWait time.Duration, refreshInterval time.Duration) *director {
 	d := &director{
-		uf: urlsFunc,
+		uf:              urlsFunc,
+		failureWait:     failureWait,
+		refreshInterval: refreshInterval,
```

Comment:
Where is `refreshInterval` in director struct got used? Can we remove it if it is not used anywhere?